pkg/controllers/leaderworkerset_controller.go

```diff
@@ -90,22 +91,26 @@ func (r *LeaderWorkerSetReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 
 	partition, err := r.rollingUpdatePartition(ctx, lws)
```

Comment:
I don't think these events make sense to the system, event usually generated for state transitions, or specific reasons for diagnosis, not logging. 

Please revert them. The only one makes sense to me is the `createHeadlessService` failed event, which is an important signal.

No need to created a new file to container the constants, they're not shared, located at this file is enough.

Reply:
ok, i fix it right now

Reply:
Discussed offline, hope to have a event to tell us that the rolling update is failed but there's no specific signal indicates the rolling update is failed. So maybe keep the same as sts right now.(Sts doesn't have a event to report that the rolling update is failed as well.)

Fix me if not.